test/e2e/auth/service_accounts.go

```diff
@@ -673,7 +673,15 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 	})
 
-	ginkgo.It("ServiceAccountIssuerDiscovery should support OIDC discovery of service account issuer", func() {
+	/*
+	   Release: v1.21
+	   Testname: OIDC Discovery (ServiceAccountIssuerDiscovery)
+	   Description: Ensure kube-apiserver serves correct OIDC discovery
+	   endpoints by deploying a Pod that verifies its own
+	   token against these endpoints.
+	*/
+	framework.ConformanceIt("ServiceAccountIssuerDiscovery should support OIDC discovery of service account issuer", func() {
```

Comment:
Conformance tests need to tolerate RBAC not being enabled. See for example: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/auth/certificates.go#L88.

Please fix that in another PR.

Otherwise looks OK.

Reply:
sounds good. @mtaufen, once that is opened/merged, we'll come back and ack this one

Reply:
https://github.com/kubernetes/kubernetes/pull/98813

Reply:
merged